src/Generating/StructGen.cpp

```diff
@@ -50,12 +50,21 @@ void cStructGenTrees::GenFinish(cChunkDesc & a_ChunkDesc)
 				Dest = &a_ChunkDesc;
 			}
 
-			int NumTrees = GetNumTrees(BaseX, BaseZ, Dest->GetBiomeMap());
+			int RandomNumTree = 0;
+			int NumTrees = GetNumTrees(BaseX, BaseZ, Dest->GetBiomeMap(), RandomNumTree);
 
 			sSetBlockVector OutsideLogs, OutsideOther;
 			for (int i = 0; i < NumTrees; i++)
 			{
-				GenerateSingleTree(BaseX, BaseZ, i, *Dest, OutsideLogs, OutsideOther);
+				GenerateSingleTree(BaseX, BaseZ, i, *Dest, OutsideLogs, OutsideOther, [](cNoise &, Vector3i) { return true; });
+			}
+			// generate a tree probability RandomNumTree / DividerForOnePerC
+			if (true)
```

Comment:
Is this actually supposed to have an effect - have another look at this code.

Reply:
Sorry I committed to fast